README.md

```diff
@@ -1,152 +1,330 @@
+
+#### Python requirements
+
+* Python uses the system-supplied OpenSSL. 
+* TLSv1.2 requires OpenSSL 1.0.1c or later.
+
+#### To verify your Python and TLS versions
+
+1. In a shell on your **production system**, run the command for your environment: 
 
-To check Python, in a shell on your **production system**, run: 
+    * For Python 2.x:
 
-For Python 2.x:
+        ```
+        $ python -c "import urllib2; print(urllib2.urlopen('https://tlstest.paypal.com/').read())"
+        ```
 
-`$ python -c "import urllib2; print(urllib2.urlopen('https://tlstest.paypal.com/').read())"`
+    * For Python 3.x:
 
-For Python 3.x:
+        ```
+        $ python -c "import urllib.request; print(urllib.request.urlopen('https://tlstest.paypal.com/').read())"
+        ```
 
-`$ python -c "import urllib.request; print(urllib.request.urlopen('https://tlstest.paypal.com/').read())"`
+        * On success:
+            
+            ```
+            PayPal_Connection_OK
+            ```
+        
+        * On failure, an `URLError` is raised:
+            
+            ```
+            urllib2.URLError: <urlopen error EOF occurred in violation of protocol (_ssl.c:590)>
+            urllib2.URLError: <urlopen error [Errno 54] Connection reset by peer>
+            ```
 
-- On success, `PayPal_Connection_OK` is printed.
-- On failure, an `URLError` will be raised: <br/>
-`urllib2.URLError: <urlopen error EOF occurred in violation of protocol (_ssl.c:590)>`
-`urllib2.URLError: <urlopen error [Errno 54] Connection reset by peer>`
+* * *
 
 ### Ruby
 
-Ruby 2.0.0 or above is required to use TLSv1.2 from the system supplied OpenSSL. TLSv1.2 requires OpenSSL 1.0.1c or higher. Therefore, both `Ruby > 2.0.0` and `OpenSSL > 1.0.1c` are required. You may also need to run `bundle update` to update your dependencies.
+* [Ruby requirements](#ruby-requirements)
+* [To verify your Ruby and TLS versions](#to-verify-your-ruby-and-tls-versions)
 
-*For the PayPal legacy Ruby SDK (packaged as `PP_Ruby_NVP_SDK.zip`), please download [this update](https://github.com/paypal/TLS-update/blob/master/ruby/PP_Ruby_NVP_SDK.zip).*
+#### Ruby requirements
 
-To check Ruby, in a shell on your **production system**, run:
+* Ruby 2.0.0 or later and OpenSSL 1.0.1c or later are required:
+    
+    * Ruby 2.0.0 or later is required to use TLSv1.2 from the system-supplied OpenSSL.
+    * TLSv1.2 requires OpenSSL 1.0.1c or later.
 
-`$ ruby -r'net/http' -e 'puts Net::HTTP.get(URI("https://tlstest.paypal.com/"))'`
+* To update your dependencies, you might need to run `bundle update`.
 
-- On success, `PayPal_Connection_OK` will be printed.
-- On failure, a `OpenSSL::SSL::SSLError` or `EOFError` will be thrown.
+#### To verify Ruby and TLS versions
 
-### Node.js
+1. For the PayPal legacy Ruby SDK packaged as `PP_Ruby_NVP_SDK.zip`, download this [PP_Ruby_NVP_SDK.zip](https://github.com/paypal/TLS-update/blob/master/ruby/PP_Ruby_NVP_SDK.zip).
```

Comment:
This is really a special callout that should probably be in its own section (e.g. something like `PayPal Legacy Ruby SDK Update`) and not part of the verify section.